dev-docs/bidders/gumgum.md

```diff
@@ -42,3 +42,7 @@ information.
 | `video`        | optional | Tracking ID       | `'ggumtest'`           | `string`  |
 | `videoPubID`   | optional | Publisher ID      | `123`                  | `integer` |
 | `inVideo`      | optional | Tracking ID       | `'ggumtest'`           | `string`  |
+
+
+Please note that both video and in-video products require a mediaType of video. 
+All other products (in-screen, slot, native) should have a mediaType of banner.
```

Comment:
FYI - Prebid doesn't recommend delivering native ads through 'mediaType.banner'. If this is necessary, then we will soon require that you set meta.mediaType="native" on the bidResponse object.